charts/sonarqube-dce/README.md

```diff
@@ -130,6 +130,30 @@ Nonetheless, if you intend to run a production-grade SonarQube please follow the
 * Set `initSysctl.enabled` to **false**. This parameter would run **root** `sysctl` commands, while those sysctl-related values should be set by the Kubernetes administrator at the node level (see [here](#elasticsearch-prerequisites))
 * Set `initFs.enabled` to **false**. This parameter would run **root** `chown` commands. The parameter exists to fix non-posix, CSI, or deprecated drivers.
 
+#### Cpu and memory settings
+
+Monitoring cpu and memory is an important part of software reliability. The SonarQube helm chart comes with default values for cpu and memory requests and limits. Those memory values are matching the default SonarQube JVM Xmx and Xms values.
+
+Xmx define the maximum size of the JVM heap, this is **not** the maximum memory the JVM can allocate.
+
+This is why most recommendation state that Xmx should be set to ~80% of the total amount of memory available on the machine.(or in kubernetes, the request and limits)
+
+Please find here the default SonarQube Xmx parameter to setup the memory requests and limits accordingly.
+
+|Edition|Sum of Xmx|
+|---|---|
+|datacenter edition searchNodes|2G|
+|datacenter edition ApplicationNodes|3G|
+
+This is why to accomodate with the 80% rule we set the default to:
+
+- searchNodes.resources.memory.request/limit=3072M
+- ApplicationNodes.resources.memory.request/limit=4096M
+
+Please feel free to adjust those values to your needs but memory being a non-compresible resource, we advise to setup the memory requests and limits to the **same** value for production use case, making memory a guaranteed resource.
```

Comment:
Maybe change to ---> Please feel free to adjust those values to your needs. However, given that memory is a “non-compressible” resource, we advise you to set the memory requests and limits to the **same**, making memory a guaranteed resource. This is needed especially for production use cases.